packages/schema-dot-org-types/tools/transform-schema.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
```

Comment:
Quite true. I just didn't see the value in making a public package that generates these files.